src/main/java/io/cryostat/messaging/MessagingServer.java

```diff
@@ -95,9 +95,13 @@ public void start() throws SocketException, UnknownHostException {
 
         server.websocketHandler(
                 (sws) -> {
-                    if (!"/api/v1/command".equals(sws.path())) {
+                    if (!"/api/v1/command".equals(sws.path())
+                            && !"/api/v1/notifications".equals(sws.path())) {
                         sws.reject(404);
                         return;
+                    } else if (!"/api/v1/notifications".equals(sws.path())) {
```

Comment:
Isn't this effectively equivalent to `if ("/api/v1/notifications".equals(sws.path())` since it's an `else if` of the prior condition? I think as-is it's a little bit more confusing to read through than it needs to be, ie if someone else comes along later trying to figure out when exactly the `410` response is sent.

In other words,

```
if path is "/api/v1/command" then reject 410
else if path is not "/api/v1/notifications" then reject 404
```

seems to me like it would be more readable. What do you think?

Reply:
Definitely more readable, will change.